src/SignaturePad.Forms.Platform.Shared/SignaturePadCanvasRenderer.cs

```diff
@@ -25,6 +25,10 @@
 using Xamarin.Forms.Platform.Android;
 using NativeSignaturePadCanvasView = Xamarin.Controls.SignaturePadCanvasView;
 using NativePoint = System.Drawing.PointF;
+#elif NET471
+using Xamarin.Forms.Platform.WPF;
+using NativeSignaturePadCanvasView = Xamarin.Controls.SignaturePadCanvasView;
+using NativePoint = System.Drawing.Point;
```

Comment:
Is there a reason why we can't use `PointF`?

Reply:
Ink canvas in wpf works with Point and Size and not PointF and SizeF, I have tried to use PointF to avoid casting but I generates errors in another place

Reply:
Also, I just noticed that you are using `System.Drawing.Xxx` is it possible to use the WPF types instead? `System.Windows.Point`, `System.Windows.Size` and `System.Windows.Rect`.

I had a look at the code, and they are using the WPF `Point`, `Size` and `Rect` - which are double based.

Reply:
it must be styluspoint as default for wpf, my mistake